tapdb/sqlc/queries/assets.sql

```diff
@@ -862,7 +862,13 @@ INSERT INTO script_keys (
                          WHEN script_keys.declared_known IS NULL OR script_keys.declared_known = FALSE
                          THEN COALESCE(EXCLUDED.declared_known, script_keys.declared_known)
                          ELSE script_keys.declared_known
-                       END
+                       END,
+      -- If the tweak was previously unknown, we'll update to the new value.
+      tweak = CASE
```

Comment:
Non-blocking, but can extend the recently added TestScriptKeyKnownUpsert test to cover this

Reply:
Yeah, makes sense. Added.